workspace/notebook/py_create_lake_databases.json

```diff
@@ -86,36 +86,37 @@
-					"#raw_database_name = 'odw_raw_db'\r\n",
-					"#raw_database_location = 'abfss://odw-raw@' + storage_account\r\n",
-					"#workspace_database_name = 'odw_workspace_db'\r\n",
-					"#workspace_database_location = 'abfss://odw-workspace@' + storage_account + '/'\r\n",
-					"standardised_database_name = 'odw_standardised_db'\r\n",
-					"standardised_database_location = 'abfss://odw-standardised@' + storage_account + '/'\r\n",
-					"harmonised_database_name = 'odw_harmonised_db'\r\n",
-					"harmonised_database_location = 'abfss://odw-harmonised@' + storage_account + '/'\r\n",
-					"curated_database_name = 'odw_curated_db'\r\n",
-					"curated_database_location = 'abfss://odw-curated@' + storage_account + '/'\r\n",
-					"config_database_name = 'odw_config_db'\r\n",
-					"config_database_location = 'abfss://odw-config@' + storage_account + '/'\r\n",
-					"\r\n",
-					"curated_migration_database_name = 'odw_curated_migration_db'\r\n",
-					"curated_migration_database_location = 'abfss://odw-curated-migration@' + storage_account + '/'"
+					"logging_database_location = f\"abfss://logging@{storage_account}\"\r\n",
+					"standardised_database_location = f\"abfss://odw-standardised@{storage_account}\"\r\n",
+					"harmonised_database_location = f\"abfss://odw-harmonised@{storage_account}\"\r\n",
+					"curated_database_location = f\"abfss://odw-curated@{storage_account}\"\r\n",
+					"curated_migration_database_location = f\"abfss://odw-curated-migration@{storage_account}\"\r\n",
+					"config_database_location = f\"abfss://odw-config-db@{storage_account}\""
 				],
-				"execution_count": 2
+				"execution_count": null
 			},
 			{
 				"cell_type": "code",
+				"metadata": {
+					"jupyter": {
+						"source_hidden": false,
+						"outputs_hidden": false
+					},
+					"nteract": {
+						"transient": {
+							"deleting": false
+						}
+					}
+				},
 				"source": [
-					"#spark.sql(\"CREATE DATABASE IF NOT EXISTS {0} LOCATION '{1}'\".format(workspace_database_name, workspace_database_location))\r\n",
-					"spark.sql(\"CREATE DATABASE IF NOT EXISTS {0} LOCATION '{1}'\".format(standardised_database_name, standardised_database_location))\r\n",
-					"spark.sql(\"CREATE DATABASE IF NOT EXISTS {0} LOCATION '{1}'\".format(harmonised_database_name, harmonised_database_location))\r\n",
-					"spark.sql(\"CREATE DATABASE IF NOT EXISTS {0} LOCATION '{1}'\".format(curated_database_name, curated_database_location))\r\n",
-					"spark.sql(\"CREATE DATABASE IF NOT EXISTS {0} LOCATION '{1}'\".format(config_database_name, config_database_location))\r\n",
-					"\r\n",
-					"spark.sql(\"CREATE DATABASE IF NOT EXISTS {0} LOCATION '{1}'\".format(curated_migration_database_name, curated_migration_database_location))"
+					"spark.sql(f\"CREATE DATABASE IF NOT EXISTS logging LOCATION '{logging_database_location}'\")\r\n",
+					"spark.sql(f\"CREATE DATABASE IF NOT EXISTS odw_standardised_db LOCATION '{logging_database_location}'\")\r\n",
```

Comment:
logging_database_location is set for all databases, please change it to relevant databases

Reply:
Changed.